src/main/java/com/example/Customer.java

```diff
@@ -0,0 +1,23 @@
+package com.example;
+
+/**
+ * Public class Customer with the field loyaltyCard and
+ * constructor
+ *
+ * @author Elena Yevsyukova
+ * @version 2.1
```

Comment:
No, it's not version 2.1. You have not released this code to public ever. So it's pre 1.0. Forget about this tag for the time being.